src/test/java/uk/gov/hmcts/divorce/systemupdate/service/CaseDetailsUpdaterTest.java

```diff
@@ -18,6 +20,9 @@
 @ExtendWith(MockitoExtension.class)
 class CaseDetailsUpdaterTest {
 
+    @Spy
+    private ObjectMapper objectMapper = new ObjectMapper().findAndRegisterModules();
```

Comment:
do we still need findAndRegisterModules ?

Reply:
No need for this now, has been removed here https://github.com/hmcts/nfdiv-case-api/pull/1181/commits/8715ef34fef6acda53868f03bfd2964a49a3763c